package.json

```diff
@@ -105,6 +105,7 @@
 		"@hashicorp/react-command-line-terminal": "^2.0.4",
 		"@hashicorp/react-consent-manager": "^8.0.1",
 		"@hashicorp/react-docs-page": "^17.0.0",
+		"@hashicorp/react-docs-sidenav": "8.4.1",
```

Comment:
We have to pin this dep right now for the build to work. We can upgrade in a follow-up PR.